frontend/test/metabase/scenarios/dashboard/x-rays.cy.spec.js

```diff
@@ -0,0 +1,20 @@
+import { restore, signInAsAdmin } from "../../../__support__/cypress";
+
+describe("scenarios > x-rays", () => {
+  before(restore);
+  beforeEach(signInAsAdmin);
+
+  it("should exist on homepage when person first signs in", () => {
+    cy.visit("/");
+    cy.contains("A look at your People table");
+    cy.contains("A look at your Orders table");
+    cy.contains("A look at your Products table");
+    cy.contains("A look at your Reviews table");
+  });
+
+  it.skip("should be populated (Issue #12917)", () => {
+    cy.visit("/");
+    cy.findByText("People table").click();
+    cy.findByText("Something's gone wrong").should("not.exist");
```

Comment:
I'm guessing this test was written while it was still broken. Now that it's fixed, I think it'd be better to assert what should be there.

Reply:
@paulrosenzweig I don't think this issue is fixed yet.

Reply:
@Opalevanescence It seems to be working for me. Could you be running an old build of the frontend? We should jump on that if it's still broken.

Reply:
Ah sorry I think this was branched off master before the fix. Merging master or release-0.36.x should pull in the fix. While doing that, it's probably best to switch the base branch to release-0.36.x.

Reply:
I tried rebasing (and pulling in release-0.36.x) and rebuilt the frontend, but it still isn't working. Also, if this issue is fixed, why is #12917 still open?

Reply:
Hmm I'm not sure what's happening then. I just checked out this branch, merged in `release-0.36.x`, and x-rays are working for me. Is it the same root cause as before? Will a new dashboard with just a text card fail in the same way?

#12917 is an issue to create a test. The issue for the bug is #12873.

Reply:
Maybe it's the issue where you need to restart cypress to get the newly built frontend in the uberjar running tests? That wouldn't explain seeing the issue outside of cypress though if you're seeing that.

Reply:
Thanks for helping with getting everything running on my computer! This page is showing up correctly now.